config/manifests/4.11/aws-efs-csi-driver-operator.clusterserviceversion.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
     repository: https://github.com/openshift/aws-efs-csi-driver-operator
     createdAt: "2021-07-14T00:00:00Z"
     description: Install and configure AWS EFS CSI driver.
-    olm.skipRange: ">=4.9.0 <4.10.0"
+    olm.skipRange: ">=4.11.0 <4.11.0"
```

Comment:
Isn't this self-contradicting?

Reply:
It is :-).

Looking at LSO, the min. version should be the lowest where the operator was published, fixed.